app/models/appointment.rb

```diff
@@ -1,5 +1,7 @@
 # rubocop:disable Metrics/ClassLength
 class Appointment < ApplicationRecord
+  FAKE_DATE_OF_BIRTH = '1900-01-01'.freeze
```

Comment:
if we are moving this a more descriptive name might be nice. ```DATE_OF_BIRTH_FOR_IMPORTED_RECORDS```
might be a bit wordy however..